yarn-project/aztec.js/src/account/entrypoint/single_key_account_entrypoint.ts

```diff
@@ -18,8 +18,8 @@ export class SingleKeyAccountEntrypoint implements Entrypoint {
   constructor(
     private address: AztecAddress,
     private partialAddress: PartialAddress,
-    private privateKey: PrivateKey,
-    private signer: Signer,
+    private privateKey: GrumpkinPrivateKey,
+    private signer: Schnorr,
```

Comment:
Given this is now hardcoded to be Schnorr, not much of a point of receiving the signer as a ctor argument, right? We can probably just initialise it right when we have to sign.

Reply:
You are absolutely right. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/1919/commits/5b696fd0f80db5041bd68931240b35c482febc07